static/js/actions/index.js

```diff
@@ -13,6 +13,7 @@ export const FETCH_FAILURE = 'FETCH_FAILURE';
 export const FETCH_SUCCESS = 'FETCH_SUCCESS';
 export const FETCH_PROCESSING = 'FETCH_PROCESSING';
 
+export const REQUEST_GET_PRICES = 'REQUEST_GET_PRICES';
```

Comment:
nope, just exactly what I wrote. when the `redux-hammock` library generates a `get` function for hitting an API it sticks the action types it automatically generates on the function as properties, for convenience for testing and so on. So `actions.prices.get.requestType` is `'REQUEST_GET_PRICES'`.